mailpoet/lib/Config/Migrator.php

```diff
@@ -122,6 +124,7 @@ public function segments() {
       'deleted_at timestamp NULL,',
       'average_engagement_score FLOAT unsigned NULL,',
       'average_engagement_score_updated_at timestamp NULL,',
+      'display_in_manage_subscription_page int(1) NULL DEFAULT 0,',
```

Comment:
I think it would be nicer to use `tinyint(1)` for such a boolean flag. Using `int(1)` would still use the same size as `int(11)`. Also, we may not need to make the column nullable, I guess by using the default 0, we initially set all to 0.

Reply:
Noted